CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [4.1.2](https://github.com/webpack-contrib/image-minimizer-webpack-plugin/compare/v4.1.1...v4.1.2) (2024-12-16)
```

Comment:
Changelog is auto generated so dont modify that

Reply:
ok, i will correct it now...

Reply:
I have removed changes in `package.json` and `CHANGELOG.md`